src/MasterXaction.h

```diff
@@ -74,6 +74,8 @@ class MasterXaction : public RefCountable
     // TODO: add state from other Jobs in the transaction
 
 private:
+    friend class RefCount<MasterXaction>;
```

Comment:
> This `friend` was needed so the `RefCount::Make()` template can have access to the hidden `MasterXAction` constructor (when used by the `MasterXaction::Make*()` methods). 

Yes, I know. To avoid misunderstanding, the point of this change request is that existing MasterXaction::Make*() methods should not use the new RefCount::Make(). Instead, the existing two custom MasterXaction makers should be preserved without any changes (including changes that create compiler errors that, in turn, require adding this friend declaration).